packages/mask/src/extension/popups/components/WalletBalance/index.tsx

```diff
@@ -34,14 +33,5 @@ export const WalletBalance = memo(function WalletBalance({
         )
     }
 
-    return (
-        <Typography {...props}>
-            <FormattedBalance
-                value={balance}
-                decimals={nativeToken.decimals}
-                symbol={nativeToken.symbol}
-                formatter={formatBalance}
-            />
-        </Typography>
-    )
+    return <Typography {...props}>{`${formatBalance(balance, nativeToken.decimals)} ${nativeToken.symbol}`}</Typography>
```

Comment:
We are going to support multiple currencies. We can refactor `<FormattedBalance  />` to make it.